src/coreclr/binder/inc/defaultassemblybinder.h

```diff
@@ -16,6 +16,7 @@ class DefaultAssemblyBinder final : public AssemblyBinder
 public:
 
     HRESULT BindUsingPEImage(PEImage* pPEImage,
+        bool fOverrideAppPath,
```

Comment:
should this be `excludeAppPaths` ?

Reply:
Yes, I have changed the name mid-flight.